### ISSUE TITLE
Source Spotify, song title missing (RX-V475)

### ISSUE THREAD

**Issue by michelbalzer** [mvdwetering/yamaha_ynca#162]
I have a Yamaha RX-V475, selecting Spotify as Source the song title is missing (should be _Sabotage_):

Dev Tools State Attributes:

``` yaml
source_list:
  - ''
  - AirPlay
  - Apple TV
  - Blu-ray
  - Fire TV
  - iPod
  - iPod (USB)
  - NET RADIO
  - SERVER
  - Spotify
  - TUNER
  - TV
  - V-AUX
sound_mode_list:
  - 2ch Stereo
  - 5ch Stereo
  - Action Game
  - Adventure
  - Cellar Club
  - Chamber
  - Drama
  - Mono Movie
  - Music Video
  - Roleplaying Game
  - Sci-Fi
  - Spectacle
  - Sports
  - Standard
  - Straight
  - Surround Decoder
volume_level: 0.4690721649484536
is_volume_muted: true
source: Spotify
sound_mode: 2ch Stereo
device_class: receiver
icon: mdi:audio-video
friendly_name: Receiver
supported_features: 89533
media_artist: Gang Starr
media_album_name: The Ownerz
```

Is it not provided by the Receiver or is it just missing from in the integration?

[diagnostics file](https://github.com/mvdwetering/yamaha_ynca/files/11233301/config_entry-yamaha_ynca-6042d76b4bb9801ab0a5e20cbacaf905.json.txt)

(Tried to make a template sensor providing the played media depending on Source)

**Comment by mvdwetering** [comment]
The title shows in the diagnostics. It just seems that the Spotify source sends the title as TRACK while all other sources seem to send title as SONG. My receiver does not support Spotify so never noticed it.

I will update the integration to look at the TRACK also.

**Comment by mvdwetering** [comment]
This should be fixed in the v7.1.0 release

**Comment by michelbalzer** [comment]
Yes, fixed. Thanks!

![image](https://user-images.githubusercontent.com/1337412/234232626-6483eb7a-2737-475b-9932-a38577f39471.jpeg)
